oktapam/resource_ad_task_settings.go

```diff
@@ -413,14 +408,27 @@ func flattenADTaskSettings(taskSettings *client.ADTaskSettings) map[string]inter
 	return m
 }
 
-func createADTaskSettingsResourceID(adConnectionId string, adTaskSettingsId string) string {
-	return fmt.Sprintf("%s|%s", adConnectionId, adTaskSettingsId)
-}
-
 func parseADTaskSettingsResourceID(resourceId string) (string, string, error) {
 	split := strings.Split(resourceId, "|")
 	if len(split) != 2 {
-		return "", "", fmt.Errorf("oktapam_ad_task_settings id must be in the format of <ad connection id>|<ad task settings id>, received: %s", resourceId)
+		return "", "", fmt.Errorf("expected format: <connection_id>/<id>, received: %s", resourceId)
```

Comment:
"/" -> "|"

Reply:
Waitwait. Didn't you say you were switching to / as a divider?

Reply:
Whatever it is I just want it to be consistent.